### DISCUSSION TITLE
Welcome to WebSocket4OPC Discussions!

### DISCUSSION THREAD

**Discussion by duduyoyo** [duduyoyo/WebSocket4OPC#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ldariva** [comment]
Hi,

I wonder if you could make the dll code available.

**Reply by duduyoyo** [reply]
No plan doing that at this moment. If you want that dll code for commercial use you can let me know.

**Comment by danilNil** [comment]
Hi @duduyoyo! The tool looks promising. However I have concerns about providing admin account credentials to the tool. Is there another way of installation?

**Reply by duduyoyo** [reply]
That credential isn't used by my tool. It is used only by IIS when you configure a new app pool. I don't use it or store it anywhere else. If you study my installation script a little bit you will understand what I mean. Of course, you can go to IIS app pool to configure a new one by yourself with your desired credential, but you need modify installation script a little bit to let others go through. Cheers!

**Reply by danilNil** [reply]
@duduyoyo thank you for such a comprehensive response and a tool itself! I'm choosing between your tool and OpenOPC2.

So, I will create a PR with step-by-step guide and a modified installation script that does not require credentials if I integrate with OPC DA by using your library.

**Comment by DanilaX1** [comment]
I understand correctly that this application cannot run under Windows 7. Since OPC is currently installed on a machine with Windows 7. I need to install an OPC server on Windows 10. To run your server?

**Reply by duduyoyo** [reply]
It is correct that you need install OPC server in a version which can support WebSocket, so it must be Windows 8 (or server 2012) and above. Then you can put my application there. Good luck and let me know if you need any help!
